packages/test/test-service-load/src/utils.ts

```diff
@@ -154,14 +154,17 @@ export async function initialize(testDriver: ITestDriver, seed: number, testConf
     const testId = Date.now().toString();
     const request = testDriver.createCreateNewRequest(testId);
     await container.attach(request);
+    assert(container.resolvedUrl !== undefined, "Container missing resolved URL after attach");
+    assert(container.resolvedUrl.type === "fluid", "Container has a non-fluid resolved URL after attach");
+    const containerId = container.resolvedUrl.id;
     container.close();
 
     if ((testConfig.detachedBlobCount ?? 0) > 0) {
         // TODO: #7684 this should be driver-agnostic
         const url = (testDriver as OdspTestDriver).getUrlFromItemId((container.resolvedUrl as any).itemId);
         return url;
     }
-    return testDriver.createContainerUrl(testId);
+    return testDriver.createContainerUrl(containerId);
```

Comment:
i do not believe this is the correct fix. it will not work for odsp.

Reply:
elsewhere it is done like this:
```suggestion
    return testDriver.createContainerUrl(testId, container..resolvedUrl);
```